.gitattributes

```diff
@@ -1,6 +1,8 @@
-/tests/*.php export-ignore
-/docs export-ignore
+/tests export-ignore
```

Comment:
This breaks the composer autoload classmap for `tests/behat/` as `tests/` doesn't exist.

Reply:
There are no classes there so we should remove that entire directive.